env/config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Because it is a localhost configuration, we can rename the file to config.local.json

Reply:
But we can do this later as well

Reply:
Yes in this way we need another file for production. Maybe later do it

Reply:
Yep